cost/google/cloud_sql_idle_instance_recommendations/google_sql_idle_instance_recommendations.pt

```diff
@@ -341,52 +338,69 @@ script "js_sql_instances_cost_mapping", type:"javascript" do
+        database["resourceLink"] = resource_link
+    })
+
+    _.each(recommendations, function (recommendation) {
+        count ++
+        var rec_resource_link = recommendation.resourceLink.replace("//compute.googleapis.com/", "")
+        var database = _.find(databases, function(db) {
+            return db.resourceLink == rec_resource_link
+        })
+        if (!(database == undefined || database == null)) {
+            count++
+            if (recommendation['currency'] !== undefined) {
+                if (ds_currency_reference[recommendation['currency']] !== undefined) {
+                    cur = ds_currency_reference[recommendation['currency']]['symbol']
                 } else {
-                    var cur = "$"
-                    var separator = ","
-                }
-                if (recommendation.costNanos < 0) {
-                    recommendation.costNanos = recommendation.costNanos * -1
-                }
-                if (recommendation.costUnits < 0) {
-                    recommendation.costUnits = recommendation.costUnits * -1
+                    cur = "$"
                 }
-                var strNano = "." + recommendation.costNanos
-                var combine = recommendation.costUnits + parseFloat(strNano)
-                var monthsDuration = parseFloat(recommendation.duration.replace("s", "")) / 2628288
-                var monthlySavings = combine / monthsDuration
-                total = total + monthlySavings
-                database['savings'] = (Math.round(monthlySavings * 1000) / 1000)
                 database['savingsCurrency'] = cur
-                database['priority'] = recommendation['priority']
-                database['primaryImpact'] = recommendation['primaryImpact']
-                database['recommenderSubtype'] = recommendation['recommenderSubtype']
-                database['state'] = recommendation['state']
-                database['region'] = recommendation['region']
-                database['service'] = "SQL"
-                var tags = []
-                if (database['tags'] != null) {
-                    Object.keys(database.tags).forEach(function (key) {
-                        tags.push(key + '=' + database.tags[key])
-                    });
-                }
-                database['tags'] = tags
-                instances.push(database)
+            } else {
+                cur = "$"
+                var separator = ","
             }
-        })
+
+            //Convert Costs from Negative to Positive and add Units and Nanos
+            if (!(recommendation.costNanos == null || recommendation.costNanos == undefined)) {
+              if (recommendation.costNanos != 0) {
+                recommendation.costNanos = recommendation.costNanos * -1
+              }
+            } else {
+              recommendation.costNanos = 0
+            }
+
+            if (recommendation.costUnits < 0) {
```

Comment:
```suggestion
            if (recommendation.costUnits != 0) {
```

Reply:
@midir99 sorry for this one, I just fixed it.